### PR TITLE
add new description for alb002 and consistent description alb003

### PR DESCRIPTION
**Pull request by Ghesselink**
A new description is added for ALB002 and the description of ALB003 is now consistent with the other descriptions. 

Rules that don't have a description yet:

- GEM001
- GEM002
- SPS001

### CODE REVIEW

File: features/ALB003_Alignment-directions.feature [hunk 1]
```diff
@@ -1,11 +1,8 @@
 @implementer-agreement
 @ALB
 Feature: ALB003 - Alignment Nesting
```
Comment (pjanck) [inline features/ALB003_Alignment-directions.feature:3]: File name disagrees.

File: features/ALB002_Alignment-layout.feature [hunk 2]
```diff
@@ -1,6 +1,8 @@
 @implementer-agreement
 @ALB
 Feature: ALB002 - Alignment Layout
+This rule that alignment has a nesting relationship with its layout components, and vice versa. Furthermore, the layouts
```
Comment (pjanck) [inline features/ALB002_Alignment-layout.feature:4]: Formulation needs some attention, there seems something missing.

**Review by pjanck — CHANGES_REQUESTED** [review]

File: features/ALB003_Alignment-directions.feature (hunk 1 above)
Comment (Ghesselink) [reply features/ALB003_Alignment-directions.feature]: Thanks for noticing. It's fixed, also added a test for it checks this automatically 
https://github.com/buildingSMART/ifc-gherkin-rules/commit/52d2a6f2aac9f7e5f669e39edf31165273695eb7

**Review by aothms — CHANGES_REQUESTED** [review]

File: features/environment.py [hunk 3]
```diff
@@ -2,10 +2,32 @@
...
 
+from pyparsing import Word, alphas, alphanums, delimitedList, OneOrMore
+
 def before_feature(context, feature):
     # @tfk we have this strange issue between stack frames blending over
     # between features so we need to preserve only the bottom two stack
     # frames when beginning a new feature.
+
+    # parse filename
+    tree = delimitedList(Word(alphas + '/'), '/')
```
[the remark this answers is not in the document]
Comment (Ghesselink) [reply features/environment.py]: I've changed it in https://github.com/buildingSMART/ifc-gherkin-rules/pull/43/commits/4db4722af5d76dbfc53495230f25d1cf80f6237e

One thing is, in the filename different special symbols are used. This one takes that into consideration when comparing. What do you think?

```
# ffn = 'ALB001_Alignment-in-spatial-structure.feature'
    file_feature_name = os.path.basename(context.feature.filename).replace('.feature', ' ') #'ALB001_Alignment-in-spatial-structure '
    file_feature_name = ''.join(c for c in file_feature_name if c.isalnum()) #'ALB001Alignmentinspatialstructure'
    # gherkin_feature_name  = 'ALB001 - Alignment in spatial structure'
    gherkin_feature_name = ''.join(c for c in feature.name if c.isalnum()) #'ALB001Alignmentinspatialstructure'
    
    assert file_feature_name.lower() == gherkin_feature_name.lower() # True
```

**Comment by Ghesselink** [comment]
Close due to the new PR https://github.com/buildingSMART/ifc-gherkin-rules/pull/59 and the already-merged PR https://github.com/buildingSMART/ifc-gherkin-rules/pull/53

**Pull request closed without merging**